mantle/cmd/kola/options.go

```diff
@@ -197,6 +193,10 @@ func syncOptionsImpl(useCosa bool) error {
 		return fmt.Errorf("oscontainer is only supported on rhcos")
 	}
 
+	if kola.Options.Distribution == "" {
+		kola.Options.Distribution = kolaDistros[0]
+	}
```

Comment:
Should we do something like this here?

```golang
} else {
   	if err := validateOption("distro", kola.Options.Distribution, kolaDistros); err != nil {
		return err
	}
}
```

That way we still check that it's a valid distro when it's explicitly provided.